docs/blog/crucial-tests/index.md

```diff
@@ -394,6 +392,50 @@ beforeAll(() => {
 });
 ```
 
+## 📦 Test the package as a consumer
+
+a lot of times we don't test our packages as how our consumers will use them,
+this is a simplified example of how we could do that
+
```

Comment:
Before the working example, I would consider showing the failing example briefly:

// package.json, only the files entry with index.js only

// index.js file which require 'lib-internal.js file

// local test that require lib-internal/function, explain with comments that it will work on developer machine only

Reply:
created a failing example while guiding the reader to the trap (the mistake that happen)